openstack/compute/v2/extensions/attachinterfaces/doc.go

```diff
@@ -18,5 +18,14 @@ Example of Listing a Server's Interfaces
 	for _, interface := range allInterfaces {
 		fmt.Printf("%+v\n", interface)
 	}
+
+Example to Get a Server's Interface
+
+	interfaceID = "0dde1598-b374-474e-986f-5b8dd1df1d4e"
```

Comment:
The interface ID seems to be the PortID. Let's change this to `portID`.

Reply:
Done: 5300702.